site/en/blog/deps-rems-97/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Deprecations and removals in Chrome 97
+description: >
+  A round up of the deprecations and removals in Chrome 97 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2021-11-18
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/8hNVkXWdaC9NGJ5Pg0HT.png'
+alt: >
+  Deprecations and Removals hero logo
+tags:
+  - deprecations
+  - removals
+  - chrome-97
+---
+
+{% include 'partials/see-all-dep-rem.md' %}
+
+Chrome 97 beta was released on November 18, 2021 and is expected to become the
+stable version in late December, 2021.
+
+## Remove SDES Key Exchange for WebRTC
+
+The SDES key exchange mechanism for WebRTC has been declared a MUST NOT in the
```

Comment:
@rachelandrew What does this mean? Should we clarify?

Reply:
It means it's prohibited - https://www.ietf.org/rfc/rfc2119.txt so I think this is probably fine. We're essentially just backing up why it's being removed.